rac-gcp-deploy/executor/Dockerfile

```diff
@@ -2,27 +2,15 @@ ARG GCLOUD_SDK_VERSION
 FROM google/cloud-sdk:${GCLOUD_SDK_VERSION:-304.0.0-alpine}
 
 ARG SBT_VERSION
-ENV SBT_VERSION ${SBT_VERSION:-1.4.2}
+ENV SBT_VERSION ${SBT_VERSION:-1.4.7}
 
 RUN apk --update add openjdk8 jq
```

Comment:
Is this where we can upgrade the Java version? I saw yesterday that parts of the latest FS2 release are compiled with Java 11, which failed a build, so now might be a time to upgrade

Reply:
Update: they seem to have published a fix for that issue, but I wonder whether we have a plan for this? I've wanted to use some Java 9 APIs a couple of times and got burned once the code reached CircleCI

Reply:
yes, this is the place.
I am not sure if compiling with java 11 will work for all of our services, but I believe it should be fine.
We can upgrade it and see how it goes or we can compile locally all of our services with java11 before hand to be on the safe side. :D

Reply:
Given the uncertainty, I will leave this for another PR